### DISCUSSION TITLE
How to show tabs length as 2 spaces?

### DISCUSSION THREAD

**Discussion by zer09** [jesseduffield/lazygit#3379] [Q&A]
Most of my codes using tabs instead of spaces. And it look like `lazygit` tab is equivalent to 4 spaces.
How I can set up to show tabs as 2 spaces?
I cant find it on the config

**Comment by mark2185** [comment]
Does `git diff` from the terminal expand it correctly?

**Comment by stefanhaller** [comment]
When using `git diff` or `git show` in the terminal, the tab size is determined by the terminal, or by options passed to less if you are using less as a pager. See https://stackoverflow.com/a/39352670.

In lazygit (in gocui, actually), the tab size is currently [hard-coded to 4](https://github.com/jesseduffield/gocui/blob/86e0d5a312de1ba0b89435584fc990beb53448b2/view.go#L800). It shouldn't be hard to make this configurable.
